src/storage/memory_cache/tests.rs

```diff
@@ -372,6 +372,8 @@ async fn test_store_write_back() {
     let mut block = Block::new_zeroed(BLOCK_SIZE_IN_BYTES);
     block.set_dirty(true);
     cache.store(0, 0, block).await.unwrap();
+    // Wait for the write back task
+    tokio::time::sleep(Duration::from_millis(200)).await;
```

Comment:
Does cache.flush ensure wait for writeback finished ? If it's true,  it's better to use flush than sleep for a time (since it depends on test env)